README.md

```diff
@@ -45,6 +45,15 @@
 ```bash
 npm install vuelidate --save
 ```
+check install latest version in Vue3.0
```

Comment:
I think it would be better to point them to the `next` branch or the next docs instead? It has all the new setup and docs there.

Reply:
in https://github.com/vuelidate/vuelidate/tree/next it already has these commands